resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -31,13 +33,14 @@
     public static final int DEFAULT_WAIT_DURATION_IN_OPEN_STATE = 60; // Seconds
     public static final int DEFAULT_RING_BUFFER_SIZE_IN_HALF_OPEN_STATE = 10;
     public static final int DEFAULT_RING_BUFFER_SIZE_IN_CLOSED_STATE = 100;
+    public static final Predicate<Throwable> DEFAULT_RECORD_FAILURE_PREDICATE = (throwable) -> true;
 
     private float failureRateThreshold = DEFAULT_MAX_FAILURE_THRESHOLD;
     private int ringBufferSizeInHalfOpenState = DEFAULT_RING_BUFFER_SIZE_IN_HALF_OPEN_STATE;
     private int ringBufferSizeInClosedState = DEFAULT_RING_BUFFER_SIZE_IN_CLOSED_STATE;
     private Duration waitDurationInOpenState = Duration.ofSeconds(DEFAULT_WAIT_DURATION_IN_OPEN_STATE);
     // The default exception predicate counts all exceptions as failures.
-    private Predicate<? super Throwable> recordFailurePredicate = (exception) -> true;
+    private Predicate<? super Throwable> recordFailurePredicate = DEFAULT_RECORD_FAILURE_PREDICATE;
```

Comment:
Should be `Predicate<? extends Throwable>`. I think `super` was a mistake by me.